### ISSUE TITLE
Each download link should have a unique name associated with the name of the form they are downloading

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2512]
## Builds

- [staging](https://console.cloud.google.com/cloud-build/builds;region=global/3c5aad6c-5871-4cc6-b7d6-203d7b8c45ed?project=proudcity-1184)
- [prod](https://console.cloud.google.com/cloud-build/builds;region=global/fc716284-cc34-41fc-b962-f529293bc28a?project=proudcity-1184)

Includes:
- #2496 
- #2497 

## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/recAiaSN3DfYhelg2?blocks=hide

## Describe the bug

All of the download links on the page have the same name: Download.  Each should have a unique name associated with the name of the form they are downloading.  

https://www.adapacific.org/service-and-emotional-support-animals

## Recommendation

Use aria-labelledby to associate the link with the name of the form/heading.  Give the name of the form an id to reference.   See the following reference: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA7

## Screenshots

![download link](https://github.com/proudcity/wp-proudcity/assets/1569463/54ca59a5-2241-4078-a1ed-eceb447dd5fe)

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell The changes for the issues below are up on Beta for you to look at. You can see all widgets [here](https://beta.proudcity.com/test-new-menu-and-widgets/). We edited the Events widgets and all Document widgets with a download button. I'm pushing this out to all `test` sites now.

- #2496 
- #2497  
- this issue

**Comment by kevindherman** [comment]
@curtismchale @lukefretwell One piece of event feedback, I update: https://github.com/proudcity/wp-proudcity/issues/2496

For this issue, I'm not seeing the change on beta:

List view still shows previous icon: https://beta.proudcity.com/test-new-menu-and-widgets/
![Screenshot 2024-08-20 at 4 40 33 PM](https://github.com/user-attachments/assets/d4cf77fa-81f4-4508-a73c-83beb3994790)

Document: https://beta.proudcity.com/documents/pc-packet-2018-12-11/
![Screenshot 2024-08-20 at 4 43 21 PM](https://github.com/user-attachments/assets/b0bccaff-2b48-49cc-8436-453aa6a742ce)

**Comment by curtismchale** [comment]
Deployed

Composer: https://github.com/proudcity/wp-proudcity/releases/tag/2024.08.20.1153
Build: https://github.com/proudcity/proud-recipes/releases/tag/2024.08.20.1153

**Comment by curtismchale** [comment]
@kevindherman it should use the previous icon. We're using the html tag `aria-labelledby` which provides a number that corresponds to the `id` of the title of the download. That tells screen readers that the label to refer to for the Download button is the title.

There should be no visual change.
